python2-pybind11/setup.py

```diff
@@ -3,7 +3,7 @@
 from distutils.core import setup, Extension
 from distutils import sysconfig
 
-cpp_args = ['-std=c++14', '-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
C++14 is strongly recommended because it will produce much smaller binaries. -fvisibility=hidden is even more important to achieve a small binary size (see the FAQ in the docs).